lib/mapbox/geocode.py

```diff
@@ -77,7 +77,7 @@ def main(argv=None):
         elif not is_geocode and len(args) == 2:
             latitude = args[0]
             longitude = args[1]
-            address = reverse_geocode(latitude, longitude)
+            address = reverse_geocode(latitude, longitude)['address']
```

Comment:
No, just return the whole dictionary/as much data as possible, normalized to what various API providers might return.

In practice, we might use several, not just MapBox.